### PR TITLE
Preparing release 0.1.0

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by mansimarkaur** [comment]
Should we wait for validating the 400 error responses first before releasing?

**Comment by Natim** [comment]
We can release multiple times and I am confident that it should be fine with the other errors. But we can wait for sure.

**Review by mansimarkaur — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -62,7 +62,7 @@ build-requirements:
 	$(VIRTUALENV) $(TEMPDIR)
 	$(TEMPDIR)/bin/pip install -U pip
 	$(TEMPDIR)/bin/pip install -Ue .
-	$(TEMPDIR)/bin/pip freeze > requirements.txt
+	$(TEMPDIR)/bin/pip freeze | grep -v -- '^-e' > requirements.txt
```
Comment (mansimarkaur) [inline Makefile:65]: Can you please explain this change to me?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
+appdirs==1.4.0
```
Comment (mansimarkaur) [inline requirements.txt:1]: Did you generate all of these using some command?

Comment (Natim) [reply requirements.txt:1]: Yes using the `make build-requirements` command.

File: Makefile (hunk 1 above)
Comment (Natim) [reply Makefile:65]: This is a pip freeze to get all the lib version that were installed at the time of the release.
The `grep` command is to not keep the git link of the dependencies (webpush-channels one in this case)

**Comment by mansimarkaur** [comment]
I think we should merge this now. @Natim ?

**Comment by Natim** [comment]
Let's do the release together I will show you how it's done. Please create a pypi account so that you can release yourself.

**Comment by mansimarkaur** [comment]
I have a pypi account. Let's do the release :D

**Comment by mansimarkaur** [comment]
Should I add this in the CHANGELOG - Ensured JSON responses for all requests?

**Comment by Natim** [comment]
> Should I add this in the CHANGELOG - Ensured JSON responses for all requests?

You could but for me it is already part of this changelog:

 - Add the subscription management API (#10)
- Add the channel registration manageent API (#12)

If these two items where parts of a previous release we would.

**Comment by Natim** [comment]
If you want to be prepared for the release, there is the flow (that we should add to the documentation)

Install `zest.releaser`

**Step 1**
- [x] Update changelog
- [x] Update version in `docs/source/conf.py`
- [x] Version dependencies: `make build-requirements`

`run: prerelease`

**Step 2**

`git checkout master`
`git merge --no-ff prepare-0.1.0`
`run: release`

- [x] Tag vX.Y.Z
- [x] Publish on Pypi

`run: postrelease`

**Step 3**
- [x] Add entry in Github release page
- [x] Tweet!
- [ ] Deploy new version on demo server

**Pull request merged**